### DISCUSSION TITLE
Regional Prompter - Anyone Have it Working in Vladomatic?

### DISCUSSION THREAD

**Discussion by fcabanski** [vladmandic/sdnext#1351] [Q&A]
https://github.com/hako-mikan/sd-webui-regional-prompter

I installed it but didn't use it yet. Up until Tuesday, there had been no problem. Tuesday, Vladomatic was crashing, and Vladomatic (the guy) correctly pointed out it was an extension issue, not an issue with Vladomatic.

Removing the Regional Prompter extension (deleting the dir) solved the issue: Vladomatic now loads and runs fine.

I did enter an issue on the Regional Prompter github. However, I wonder if anyone has it working in Vladomatic, found some fix or work around? Someone told me it's working in A1111, but I like Vladomatic a lot better, so don't want to go back.

**Comment by Machinekng** [comment]
I've been using it for gens for the past few days. I haven't used it prior, so it's possible that behavior isn't optimal, but I was able to replicate the example in the extension GitHub page fine and it seems to be working. It doesn't seem to be causing any crashes.

**Comment by Davikar** [comment]
Same. Have been using it for a while without problems.
Although the Latent generation mode doesn't seem to be working, but I guess that's an issue with the extension itself.

**Comment by fcabanski** [comment]
The folks at the project made an update which fixed the issue it was having on my PC. Others have said it was always running fine for them in Vladomatic. 

I used it today for the first time. It's pretty good, but trying to make multiple people, one in each zone, doesn't do very well. Maybe there are settings.

If I do elephant BREAK man, I get an elephant in one zone and man in the other. If I do man in suit BREAK man in shorts, I get a single man, maybe in shorts that seem to have suit pant material.
